Makefile

```diff
@@ -43,10 +71,74 @@ publish-pattern:
         PATTERN_NAME="$(PATTERN_NAME)" \
 	hzn exchange pattern publish -f pattern.json
 
+publish-deployment-policy:
+	@echo "============================"
+	@echo "PUBLISHING DEPLOYMENT POLICY"
+	@echo "============================"
+	@hzn exchange deployment addpolicy -f deployment.policy.json $(HZN_ORG_ID)/policy-$(SERVICE_NAME)_$(SERVICE_VERSION)
+	@echo ""
+
+remove-deployment-policy:
+	@echo "=========================="
+	@echo "REMOVING DEPLOYMENT POLICY"
+	@echo "=========================="
+	@hzn exchange deployment removepolicy -f $(HZN_ORG_ID)/policy-$(SERVICE_NAME)_$(SERVICE_VERSION)
+	@echo ""
+
+
+agent-run:
+	@echo "================"
+	@echo "REGISTERING NODE"
+	@echo "================"
+	@hzn register --policy=node.policy.json
+	@watch hzn agreement list
+
+agent-run-pattern:
+	@hzn register --pattern "${HZN_ORG_ID}/$(PATTERN_NAME)"
+
+run: stop
+	docker run -d \
+        --name ${SERVICE_NAME} \
+        --restart unless-stopped \
+        -p 8000:8000 \
+        $(SERVICE_CONTAINER)
+test:
+	curl -sS localhost:$(PORT)/
+
+push:
+	docker push $(DOCKER_HUB_ID)/$(NAME):$(VERSION)
+
+stop:
+	@docker rm -f $(NAME) >/dev/null 2>&1 || :
+
+clean: stop
+	-docker rmi $(DOCKER_HUB_ID)/$(NAME):$(VERSION) 2>/dev/null || :
```

Comment:
Is `DOCKER_HUB_ID` initialized at the top of the Makefile?  I see it referenced in the README but I don't see it here in the diff for the Makefile.

Reply:
Yes it is already initialized to "ibmosquito".